x-pack/metricbeat/module/mssql/performance/_meta/docs.asciidoc

```diff
@@ -2,11 +2,19 @@
 
 We fetch the following data:
 
-* Page splits per instance: Cumulative per instance. Show diffs between periodic readings to identify periods of frequent page splits.
-* Page life expectancy in seconds: The expected time in seconds that a data page will remain in the buffer pool.
-*  Lock wait time in seconds: Cumulative per instance. Show diffs between periodic readings to identify periods of high lock contention.
-* Total number of user connections.
-* Cumulative (per instance) recompilations time in seconds: Show diffs between periodic readings to identify periods of high SQL re-compilations.
-* Compilations time in seconds: Cumulative per instance. Show diffs between periodic readings to identify periods of high SQL compilations.
-* Batch requests time in seconds: Cumulative per instance. Show diffs between periodic readings to identify periods of high request activity.
-* Buffer Cache hit: Percentage of data pages found in buffer cache without having to read from disk.
+* *Page splits per instance*: Cumulative per instance. Show diffs between periodic readings to identify periods of frequent page splits.
```

Comment:
I really like these descriptions. It would be nice if in the future we could directly generate this list out of the fields.yml to have it only in one place.

Reply:
Actually that's a very cool idea for a script... :+1: